rfcs/0000-kuadrant-ci-cd.md

```diff
@@ -0,0 +1,837 @@
+git tag -a v0.5.0 -m "[tag] Kuadrant Operator v0.5.0"
+git push origin v0.5.0
+```
+
+4. Run the GHA 'Build Images' with the following parameters:
+* Branch containing the release workflow file – example: `release-0.5.0`, default: `main`
+* Operator bundle version (without prefix) - example: `0.5.0`, default: `0.0.0`
+* Operator tag (with prefix) - example: `v0.5.0`, default: `latest`
+* Authorino Operator bundle version (without prefix) - example: `0.10.0`, default: `latest`
+* Limitador Operator bundle version (without prefix) - example: `0.7.0`, default: `latest`
+* WASM Shim version (without prefix) – example: `0.3.0`, default: `latest`
+* Kuadrant operator replaced version (without prefix) - example: `0.4.1`, default: `0.0.0-alpha`
+* Bundle and catalog channels (comma-separated) – example: `stable`, default: `preview`
+
+
+### OperatorHub
+The current 3 operators (Authorino, Limitador and Kuadrant) are published to [OperatorHub](https://operatorhub.io/).
+This is done manually, copying the released bundle from each operator and opening a PR to the [community-operators](https://github.com/k8s-operatorhub/community-operators/)
+and [community-operators-prod](https://github.com/redhat-openshift-ecosystem/community-operators-prod/) repositories.
+The documentation for this process can be found in the [contribution section of OperatorHub](https://operatorhub.io/contribute).
+
+A brief summary of the process is as follows:
+
+1. In each community-operators repository, create a branch named after the name of the operator and its version, following
+the template `[operator-name]-[semantic-version]`, e.g.`kuadrant-operator-v0.5.0` and create a directory within the
+`operators/[operator-name]/[semantic-version]` path.
+```sh
+git checkout -b kuadrant-operator-v0.5.0
+mkdir -p operators/kuadrant-operator/0.5.0
+```
+
+2. Copy the bundle and bundle.Dockerfile from the released operator branch to the `community-operators` repository directory just created.
+```sh
+cp -r [path-to]/kuadrant-operator/bundle/* [path-to]/community-operators/operators/kuadrant-operator/0.5.0/
+cp [path-to]/kuadrant-operator/bundle.Dockerfile [path-to]/community-operators/operators/kuadrant-operator/0.5.0/
+```
+
+3. Create a PR to the `community-operators` (same for `community-operators-prod`) repository and follow the guidelines
+exposed in the auto generated description of the PR.
+
+</details>
+
+## New process
+The new process ideally should be as simple as possible, and should be able to be triggered by any person, process and
+set of events (particular commits, tags, merge, etc) that needs the artifacts. It also should be able to be repeated
+in a consistent way and without the need of any human setting variables or parameters at the time of triggering it
+(at least for the most common cases).
+
+Given our current process, we can identify some similarities and particularities between the different components:
+
+![](0000-kuadrant-cicd-release/kuadrant-components.png)
```

Comment:
I think these components are outdated, both MGC and DNS operator are missing

Reply:
This was on my list of things to update once the unified kuadrant PRs started merging. I can update and create a PR when i get a bit of time.

Reply:
@mikenairn That'd be great! thanks 🙏🏼